debian/compat

```diff
@@ -1 +1 @@
-9
+11
```

Comment:
I'm not familiar with this but it's a bit confusing that the PR description states
>bumping to deb compat 12
but its actually changed to 11

Reply:
I've reworded PR description. It was correct in commit message.